docs/pages/becoming_a_maker/overview.mdx

```diff
@@ -0,0 +1,274 @@
+import { Callout } from 'nextra/components'
+
+## An Overview of Becoming a Maker
+
+Makers run an automated swap backend (`asb`), which users can connect to and swap with.
+The `asb` accepts Bitcoin and sells Monero, for a fee.
+
+The `asb` needs to communicate with the Bitcoin and Monero blockchains.
+For this, it uses `monero-wallet-rpc` for Monero and `electrs` for Bitcoin.
+
+It's also strongly recommended to run your own Monero and Bitcoin nodes.
+
+## Docker Compose setup
+
+We maintain a Docker Compose configuration ([link](https://github.com/UnstoppableSwap/asb-docker-compose)) that automatically starts and manages these services:
+
+- `asb` (the maker service, connecting to `monero-wallet-rpc` and `electrs`)
+- `monero-wallet-rpc` (an RPC server, managing the Monero wallet and connecting to `monerod`)
+- `electrs` (a Bitcoin blockchain indexer, connecting to `bitcoind`)
+- `monerod` (a Monero node, connecting to the Monero blockchain)
+- `bitcoind` (a Bitcoin node, connecting to the Bitcoin blockchain)
+
+To run this setup you'll need Docker and Docker Compose installed.
+
+### Getting started
+
+Now you can clone the configuration repo and `cd` into it.
+We're going to setup an asb on mainnet.
+If you want to setup for testnet, go into the `testnet` directory instead.
+All other steps remain the same.
+
+```bash
+git clone https://github.com/UnstoppableSwap/asb-docker-compose.git
+cd asb-docker-compose/mainnet
+```
+
+The directory contains three files: the docker compose file, the enviroment variables file and the asb configuration file.
+The directory structure looks like this:
+
+```bash
+asb-docker-compose/mainnet/
+├─ config_mainnet.toml  # asb configuration
+├─ docker-compose.yml   # docker compose configuration
+├─ .env                 # port configuration for docker compose
+```
+
+The `docker-compose.yml` and `.env` files are part of the docker compose setup.
+We will prioritize the asb configuration file in this guide, because you probably don't want to change the docker compose setup.
+
+### Usage and commands
+
+_This list is also available in the [repository](https://github.com/UnstoppableSwap/asb-docker-compose/), including variations for testnet._
+
+First, make sure you're in the directory with the `docker-compose.yml` file:
+
+```bash copy
+cd asb-docker-compose/mainnet
+```
+
+If you aren't familiar with docker compose, here are the most important commands:
+
+| Command | Description |
+| --- | --- |
+| `docker compose up -d` | Start all services. |
+| `docker compose down` | Stop all services. |
+| `docker compose pull` | Pull the latest images for all services. You need to run `docker compose up -d` after this to actually update the services. |
+| `docker compose logs -f` | Access the logs of all services. To only see the last e.g. 100 lines, use `docker compose logs -f | tail -n 100`. |
```

Comment:
docker compose logs -f --tail 100

Reply:
Also add an example for displaying the logs of a single service 

docker compose logs -f mainnet_asb